### DISCUSSION TITLE
How to interpret Model Coefficients

### DISCUSSION THREAD

**Discussion by Snowcatcat** [facebookexperimental/Robyn#291] [Q&A] [answered]
Hi,
I'm looking at the model coefficients (demo) and finding that they are pretty large compared to what we normally would get.

Digital_I 39878.361
OLV_I 11370.256

I know the media variables have been transformed and do not have linear relationships with the response, but I'm not sure what would be an appropriate way to interpret these coefficients. Is there a way to find each channel's contribution at the weekly level?

**Also, from the CSV file, what are roi_mean, roi_total, and mean_response?**

      rn               coef          mean_spend      mean_response       roi_mean        total_spend    total_response     roi_total             solID
1: Digital_I   39878.361        337618.08          18866.197            0.055880293      4051417        319732.39        0.07891866        2_184_3
2:  OLV_I      11370.256          87266.61           2221.274             0.025453888      1832599         70239.29         0.03832770         2_184_3

**Comment by gufengzhou** [comment, marked as answer]
Hi, first of all, you can find the schema of all csv [here](https://github.com/facebookexperimental/Robyn/blob/main/demo/schema.R).

In the _alldecomp.csv, you can find the predicted value for each independent variable as time series, which is what you're asking for as "weekly contribution".

Finally, all independent variables are transformed before the fitting as you correctly acknowledged. Because the hill function we use for saturation transformation normalises all independent to 0-1, that's why beta coefficients gets bigger.

**Reply by Snowcatcat** [reply]
Thank you so much!
